python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -466,7 +466,8 @@ def flush(self):
 
     def _setFlushed(self):
         """ Like Node only descend into objects which aren't in the Schema"""
-        for index in self._cachedJobs.keys():
-            self._cachedJobs[index]._setFlushed()
+        def flushFunc(elem):
+            elem._setFlushed()
+        map(flushFunc, self._cachedJobs.items())
```

Comment:
I'm not sure this is a good use of `map`. `map` should be used to *map* one sequence onto another, not as a replacement for a `for` loop. Why not:
```python
for elem in self._cachedJobs.items():
    elem._setFlushed()
```
?